src/invidious.cr

```diff
@@ -399,512 +402,18 @@ Invidious::Routing.get "/s_p/:id/:name", Invidious::Routes::Images, :s_p_image
 Invidious::Routing.get "/yts/img/:name", Invidious::Routes::Images, :yts_image
 Invidious::Routing.get "/vi/:id/:name", Invidious::Routes::Images, :thumbnails
 
+Invidious::Routing.get "/modify_notifications", Invidious::Routes::Notifications, :modify
+
+Invidious::Routing.post "/subscription_ajax", Invidious::Routes::Subscriptions, :toggle_subscription
+Invidious::Routing.get "/subscription_manager", Invidious::Routes::Subscriptions, :subscription_manager
```

Comment:
Shouldn't those routes also be excluded for api_only builds?

Reply:
I'm not familiar with that, so... should these routes also be excluded for api_only builds?

Reply:
Yes

Reply:
After thinking a bit about that, yeah, they should be excluded.